.github/workflows/knative-style.yaml

```diff
@@ -119,16 +130,6 @@ jobs:
 
           echo "${TEMP_PATH}" >> $GITHUB_PATH
 
-      - id: golangci_configuration
-        uses: andstor/file-existence-action@v1
-        with:
-          files: .golangci.yaml
-      - name: Go Lint
-        if: steps.golangci_configuration.outputs.files_exists == 'true'
-        uses: golangci/golangci-lint-action@v2
-        with:
-          version: v1.42
```

Comment:
Did you mean to drop this?

Reply:
I blindly copied the latest actions from `knative-sandbox/.github` so I guess it was dropped there?